app/controllers/invoices_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
   def show
     @invoice = Invoice.find(params[:invoice_id])
     @items = @invoice.items
```

Comment:
I don't think this is new code on this pull request but just tagging this. Should we try to only have one instance variable here?